### ISSUE TITLE
Bundle custom widget lib for Qt Designer

### ISSUE THREAD

**Issue by 3nids** [lutraconsulting/qgis-mac-packager#80]
It would be nice to have the custom widgets available for Qt Designer

1) Run cmake with `-DWITH_CUSTOMWIDGETS=ON`
2) This will place the lib in `QT_PLUGINS_DIR`
3) Then the question is how any existing Qt Designer would find the lib?  
  a) The user would have to add the path to the lib before starting Qt Designer
  b) or we ship Qt Designer in the bundle?

**Comment by PeterPetrik** [comment]
thanks, I do not want to ship Qt Designer as it is quite huge, but probably user will need to find the location and add manually

**Comment by 3nids** [comment]
could we maybe just provide a launcher that sets the path and launches Qt Designer (would be found in common places)?
more info https://doc.qt.io/qtcreator/adding-plugins.html
would need further testing

**Comment by PeterPetrik** [comment]
can you explain why qgis user would need this? Is it for custom qt forms in attribute table ?

**Comment by 3nids** [comment]
either custom UI feature forms, but most likely for plugin development (use of map layer combobox for instance)

**Comment by PeterPetrik** [comment]
The repository is no longer used, because QGIS Mac packaging system is moved under OSGEO/QGIS repository https://github.com/qgis/QGIS-Mac-Packager. opened issue there
